internal/jimm/cloud.go

```diff
@@ -20,35 +20,173 @@ import (
 	ofganames "github.com/canonical/jimm/v3/internal/openfga/names"
 )
 
+// GetUserCloudAccess returns users access level for the specified cloud.
+func (j *JIMM) GetUserCloudAccess(ctx context.Context, user *openfga.User, cloud names.CloudTag) (string, error) {
```

Comment:
I think there is already a method in the permission manager that does this. Can you use that instead?